### DISCUSSION TITLE
The imported avatar does not work.

### DISCUSSION THREAD

**Discussion by HidehikoKondo** [mozilla/hubs#3798] [Hubs Cloud Questions] [answered]
The imported avatar does not work.

Not displayed on the avatar selection screen.

Errors detected on the "Featured avatars" screen: "column ._state dose not exist".

Please tell me how to solve these problems.

Repro step:
![FeaturedAvatars1](https://user-images.githubusercontent.com/3691510/99517672-fda12700-29d2-11eb-89fa-d4d0092df15a.png)
save Featured avater setting .

![FeaturedAvatars2](https://user-images.githubusercontent.com/3691510/99517675-fed25400-29d2-11eb-9f93-2d2a893c1ade.png)
error:column ._state dose not exist

![noavatar](https://user-images.githubusercontent.com/3691510/99519161-d8151d00-29d4-11eb-8e67-c4b4eb78c5f3.png)
no avatar...

**Comment by misslivirose** [comment, marked as answer]
The state in these avatar screenshots appear to be showing that the avatar was delisted, This means it won't show up in the avatar search results.

**Reply by robinkwilson** [reply]
Change this to "enabled" or "listed" (I forget which one).
![image](https://user-images.githubusercontent.com/29560735/106218045-62bf8880-618b-11eb-9f78-8321f668fd91.png)

**Reply by robinkwilson** [reply]
Marking as answered for now, let us know if you have any additional specific questions then "unmark" to get it back in the pipeline!

**Comment by xaddison** [comment]
Robin,

Thanks I would like to take you up on that info on how to revert some of the config.  I think I see the problem.  SES I think wrote over my tags and I have since updated with sendgrid updated info but it doesn’t seem to matter.
